org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/XMLTextDocumentService.java

```diff
@@ -454,7 +459,7 @@ private void triggerValidationFor(Collection<ModelTextDocument<DOMDocument>> doc
 	}
 
 	@SuppressWarnings("unchecked")
-	private void triggerValidationFor(TextDocument document) {
+	public void triggerValidationFor(TextDocument document) {

```

Comment:
For Spring XML server extension if classpath of the project changes the extension is notified of the change via (LSP protocol - command) the reaction to this would be re-resolve nsuris, re-validate files there fore etc. I noticed that XML server calls `triggerValidationFor` whenever the XML file changes. I need to do the same on the classpath change and hence made it public. Sounds like you'd want me to call this via a nicier API somehow and not just make it public?

Reply:
See https://github.com/eclipse/lemminx/pull/905#discussion_r501803334